catboost/python-package/ut/medium/test.py

```diff
@@ -11335,6 +11335,22 @@ def evaluate(self, approxes, target, weight):
 
             return error_sum, weight_sum
 
+        def gpu_evaluate(self, approx, target, weight, output, output_weight):
```

Comment:
Could you please add tests with at least 3 cases:
1. Logloss with `evaluate` only (to verify that this behavior still works)
2. Logloss with `gpu_evaluate` only (so that it would obvious from test that this function is being used)
3. Logloss with both `evaluate` and `gpu_evaluate` methods and checks the correct metric is selected on gpu and on cpu

Reply:
I added a separate test below, which checks that the right function is selected and called